### PR TITLE
Github actions restrictions

### PR DESCRIPTION
**Pull request by Shivraj0**

### CODE REVIEW

**Comment by Shivraj0** [comment]
Test workflow: https://github.com/gofynd/fdk-cli/actions/runs/6144424889/job/16669797940

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gofynd/fdk-cli",
-  "version": "4.0.0-beta.1",
```
Comment (vivek-gofynd) [inline package.json]: @Shivraj0 change this 4.0.0-beta.1 only, as there is no change related to fdk build.

File: scripts/publishCheck.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const packageJSON = require('../package.json');
```
Comment (vivek-gofynd) [inline scripts/publishCheck.js:1]: Make sure this scripts folder is not shipped with fdk final build

**Review by vivek-gofynd — CHANGES_REQUESTED** [review]

File: scripts/publishCheck.js (hunk 2 above)
Comment (Shivraj0) [reply scripts/publishCheck.js:1]: No, this is not shipped with release build bundle. We explicitly mention under `files` key in package.json which all directories should be present.

File: package.json (hunk 1 above)
Comment (Shivraj0) [reply package.json]: Okay, will revert !

**Review by vivek-gofynd — APPROVED** [review]

**Pull request merged**